### PR TITLE
Add table showing all tags to report

### PR DESCRIPTION
**Pull request by mistydemeo**

### CODE REVIEW

**Comment by Hwesta** [comment]
:+1:

Since we have a whole pane for analysis now, maybe move the tags listing to a different tab than the format info? It gets lost at the bottom of a long list.

**Comment by mistydemeo** [comment]
Updated by splitting the "report" tab into subtabs, one for "format" and one for "tags".

File: app/report/report.controller.js [hunk 1]
```diff
@@ -3,13 +3,17 @@
...
+  }]).
+
   controller('ReportController', ['$scope', 'FileList', 'SelectedFiles', function($scope, FileList, SelectedFiles) {
     $scope.records = SelectedFiles;
 
-    $scope.sort_property = 'format';
-    $scope.reverse = false;
-    $scope.sort_fn = function(record) {
-      var sort_prop = $scope.sort_property;
+    $scope.format_sort_property = 'format';
```
Comment (Hwesta) [inline app/report/report.controller.js:13]: Do we still need these if the tables are in separate tabs?  Also format table sorting appears to be broken now.

Comment (mistydemeo) [inline app/report/report.controller.js:13]: Oh heh, the tab table works - that's the one I tested. Will fix the other.

Comment (mistydemeo) [inline app/report/report.controller.js:13]: Not 100% sure how scoping works now given they're in the same controller.

Comment (mistydemeo) [inline app/report/report.controller.js:13]: Fixed.

**Comment by Hwesta** [comment]
:+1:

**Pull request merged**